pkg/broadcaster/broadcaster_test.go

```diff
@@ -0,0 +1,166 @@
+package broadcaster
+
+import (
+	"sync"
+	"testing"
+)
+
+// Add and remove Subscribers
```

Comment:
This file needs to be passed through `gofmt` (we need to enable a check on that :angel: )